pkgs/servers/unftp/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, linux-pam
+, rustPlatform
+, fetchFromGitHub
+, buildFeatures ? [ "gnu" ]
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "unFTP";
+  version = "0.14.5";
+
+  src = fetchFromGitHub {
+    owner = "bolcom";
+    repo = pname;
```

pname and repo are not considered the same and can cause issues down the path.

```suggestion
    repo = "unFTP";
```